dbms/src/Storages/DeltaMerge/ColumnFile/ColumnFileDeleteRange.cpp

```diff
@@ -20,7 +20,7 @@ namespace DB
 namespace DM
 {
 ColumnFileReaderPtr ColumnFileDeleteRange::getReader(
-    const DMContext & /*context*/,
+    const DMContextPtr & /*context*/,
```

Comment:
Why do we need to change from DMContext to DMContextPtr? Seems that a lot of file change is related with this change.